MobileWallet/TariLib/Wrappers/Contacts.swift

```diff
@@ -111,6 +132,48 @@ class Contacts {
         throw ContactsError.contactNotFound
     }
 
+    //TODO This function could be improved. Can also including pending transactions
+    func recentContacts(wallet: Wallet, limit: Int) throws -> [Contact] {
+        let (completedTxs, completedTxsError) = wallet.completedTransactions
+        guard completedTxsError == nil else {
```

Comment:
here maybe instead of guard completedTxsError == nil, we can use guard let completedTxs = completedTxs else { throw completedTxsError } so that we don't need to force unwrap completedTxs on the rest of the code